styles/native/ts/core/widgets/linechart.ts

```diff
@@ -0,0 +1,92 @@
+import { brand, font } from "../variables";
+import { LineChartType } from "../../types/widgets";
+
+/*
+
+DISCLAIMER:
+Do not change this file because it is core styling.
+Customizing core files will make updating Atlas much more difficult in the future.
+To customize any core styling, copy the part you want to customize to styles/native/app/ so the core styling is overwritten.
+
+==========================================================================
+    Line Chart
+
+    Default Class For Mendix Line Chart Widget
+========================================================================== */
+// eslint-disable-next-line @typescript-eslint/camelcase
+export const com_mendix_widget_native_linechart_LineChart: LineChartType = {
+    container: {
+        // All ViewStyle properties are allowed
+    },
+    errorMessage: {
+        // All TextStyle properties are allowed
+        fontFamily: font.family,
+        fontSize: font.sizeSmall,
+    },
+    chart: {
+        // All ViewStyle properties are allowed
+    },
+    gridAndLabelsRow: {
+        // All ViewStyle properties are allowed
+    },
+    gridRow: {
+        // All ViewStyle properties are allowed
+    },
+    gridWrapper: {
+        // All ViewStyle properties are allowed
+    },
```

Comment:
I'm not sure what the difference is between these 3 props. Do we really need them exposed? If so, how do we make it more clear what they represent?

Reply:
Agree quite confusing, what represents what. Do we need three viewstyles? Is this used for alignment?

Reply:
Removed the view styles, since you don't need to style them. The only ones left are `container` and `chart`.